app/browser/api/ledger.js

```diff
@@ -700,14 +700,21 @@ const excludeP = (publisherKey, callback) => {
   })
 }
 
-const addSiteVisit = (state, timestamp, location, tabId) => {
+const addSiteVisit = (state, timestamp, location, tabId, manualAdd = false) => {
   if (!synopsis) {
     return state
   }
 
   location = pageDataUtil.getInfoKey(location)
+
+  if (location == null) {
```

Comment:
I think that we can move this check before `location = pageDataUtil.getInfoKey(location)`

Reply:
Agreed, done